src/main/java/seedu/address/logic/pilot/linklocation/LinkPilotToLocationCommandFactory.java

```diff
@@ -0,0 +1,170 @@
+package seedu.address.logic.pilot.linklocation;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.address.commons.fp.Lazy;
+import seedu.address.commons.util.GetUtil;
+import seedu.address.logic.core.CommandFactory;
+import seedu.address.logic.core.CommandParam;
+import seedu.address.logic.core.exceptions.ParseException;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyItemManager;
+import seedu.address.model.location.Location;
+import seedu.address.model.location.PilotLocationType;
+import seedu.address.model.pilot.Pilot;
+
+/**
+ * Links a pilot to location.
+ * The location could be a place where they reside.
+ * To link a pilot to a flight, the pilot should reside in
+ * the same location as the flight's starting location.
+ *
+ */
+public class LinkPilotToLocationCommandFactory implements CommandFactory<LinkPilotToLocationCommand> {
+    private static final String COMMAND_WORD = "linklocation";
+    private static final String LOCATION_PREFIX = "/loc";
```

Comment:
Maybe should we have the prefix as /location to be consistent with the /pilot prefix? Or the other way around works too.
Overall, I think we should streamline our prefixes across all commands ie. either all use acronyms or full words - I can do this later too.